index.html

```diff
@@ -0,0 +1,97 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300;400;500;700;900&display=swap"
+        rel="stylesheet">
+    <title>Nana's portfolio</title>
+</head>
+
+<body>
+    <div class="wrapper">
+        <div class="header-wrapper">
+            <header>
```

Comment:
Hello!

I would like to know if it would be better to put the first screen (intro text and socials) in the main tag or leave it outside the main tag?

Kindly,
Nana

Reply:
Hello @rdnrn, you can leave out the `main` tag, check out this [tutorial](https://www.w3schools.com/html/html5_semantic_elements.asp)